tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/work_section_block.html

```diff
@@ -0,0 +1,10 @@
+{% load wagtailcore_tags %}
+
+<div>
+    {{ value.title }}
+</div>
+<div>
+    {% for block in value.content %}
+        {% include_block block with unique_id=block.id %}
+    {% endfor %}
```

Comment:
I feel like we should probably be doing this sort of thing in the main streamfield block template, for consistency. That way, we could always just do:
```suggestion
     {% include_block value.content %}
```